### PR TITLE
Adding 1.6 features to migration guide

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Adding `retry` and `materialize_view` items to the migration guide for v1.6

## Checklist
- [x] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/guides/migration/versions/01-upgrading-to-v1.6.md [hunk 1]
```diff
@@ -27,6 +27,13 @@ dbt Labs is committed to providing backward compatibility for all versions 1.x,
 
 ## New and changed documentation
 
+[`dbt retry`](/reference/commands/retry) is a new command that executes the previously run command from the point of failure. This convenient command enables dbt users to continue a failed command without rebuilding all upstream dependencies. 
+
+**Materialize view** support (for model and project configs) has been added for three data warehouses:
+    - [Bigquery](/reference/resource-configs/bigquery-configs)
```
Comment (matthewshaver) [inline website/docs/guides/migration/versions/01-upgrading-to-v1.6.md]:
```suggestion
    - [Bigquery](/reference/resource-configs/bigquery-configs#materialized-view)
```

File: website/docs/guides/migration/versions/01-upgrading-to-v1.6.md [hunk 2]
```diff
@@ -27,6 +27,13 @@ dbt Labs is committed to providing backward compatibility for all versions 1.x,
...
+    - [Postgres](/reference/resource-configs/postgres-configs)
```
Comment (matthewshaver) [inline website/docs/guides/migration/versions/01-upgrading-to-v1.6.md]:
```suggestion
    - [Postgres](/reference/resource-configs/postgres-configs#materialized-view)
```

File: website/docs/guides/migration/versions/01-upgrading-to-v1.6.md [hunk 3]
```diff
@@ -27,6 +27,13 @@ dbt Labs is committed to providing backward compatibility for all versions 1.x,
...
+    - [Redshift](/reference/resource-configs/redshift-configs)
```
Comment (matthewshaver) [inline website/docs/guides/migration/versions/01-upgrading-to-v1.6.md]:
```suggestion
    - [Redshift](/reference/resource-configs/redshift-configs#materialized-view)
```

File: website/docs/guides/migration/versions/01-upgrading-to-v1.6.md [hunk 4]
```diff
@@ -27,6 +27,13 @@ dbt Labs is committed to providing backward compatibility for all versions 1.x,
 
 ## New and changed documentation
 
+[`dbt retry`](/reference/commands/retry) is a new command that executes the previously run command from the point of failure. This convenient command enables dbt users to continue a failed command without rebuilding all upstream dependencies. 
+
+**Materialize view** support (for model and project configs) has been added for three data warehouses:
```
Comment (matthewshaver) [inline website/docs/guides/migration/versions/01-upgrading-to-v1.6.md]:
```suggestion
**Materialized view** support (for model and project configs) has been added for three data warehouses:
```

File: website/docs/guides/migration/versions/01-upgrading-to-v1.6.md [hunk 5]
```diff
@@ -27,6 +27,13 @@ dbt Labs is committed to providing backward compatibility for all versions 1.x,
 
 ## New and changed documentation
 
+[`dbt retry`](/reference/commands/retry) is a new command that executes the previously run command from the point of failure. This convenient command enables dbt users to continue a failed command without rebuilding all upstream dependencies.
```
Comment (runleonarun) [inline website/docs/guides/migration/versions/01-upgrading-to-v1.6.md]:
```suggestion
[`dbt retry`](/reference/commands/retry) is a new command that executes the previously run command from the point of failure. This convenient command enables you to continue a failed command without rebuilding all upstream dependencies. 
```

**Review by runleonarun — APPROVED** [review]
One tiny nit and approved!

**Review by runleonarun — APPROVED** [review]
🚢 it!

**Pull request merged**